build-aux/test-driver

```diff
@@ -0,0 +1,148 @@
+#! /bin/sh
+# test-driver - basic testsuite driver script.
```

Comment:
This file was added automatically when I ran `autoreconf -ivf` after adding a Makefile to the 't' directory. It's used when `make check` is run.